WireUI/Sources/WireIndividualToTeamMigrationUI/Views/CompletionView.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+
+struct CompletionView: View {
+
+    enum Action {
+        case goBack
+        case goToTeamManagement
+    }
+
+    let profileName: String
+    let teamName: String
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 24) {
+            Text(String.formated(key: "individualToTeam.completion.body", bundle: .module, teamName))
+                .wireTextStyle(.body1)
+            Spacer()
+
+            BackButton(
+                title: String.localized(key: "individualToTeam.completion.button.backToApp", bundle: .module),
+                action: { }
```

Comment:
we usually left `TODO: WPB-XXX` on places where something is left todo in a next PR.

Reply:
It's actually something I intend to get done in this PR

Reply:
Implemented